README.md

```diff
@@ -443,6 +443,14 @@ variables. Here's a quick guide:
 * `CLVK_FORCE_SUBGROUP_SIZE` specifies the subgroup size to use, overriding
   everything.
 
+* `CLVK_QUEUE_GLOBAL_PRIORITY` specifies the queue global priority to use if it
+  is supported by the driver:
+
+  * `0`: `VK_QUEUE_GLOBAL_PRIORITY_LOW_KHR` (default)
```

Comment:
if you prefer to have something else as default, I'm open to it, but I'll set it to low in ChromeOS, as we want compute workload to have the lowest priority to avoid slowing down the system for non-critical features.